tracing-attributes/Cargo.toml

```diff
@@ -44,6 +44,7 @@ quote = "1"
 
 [dev-dependencies]
 tracing = "0.1"
+tracing-futures = { path = "../tracing-futures", features = ["std-future"] }
```

Comment:
`std-future` feature is enabled by default, right?

Reply:
...yes, it is.

Reply:
fixed.